build/tools/PrepareRelease/GenerateIntegrationDefinitions.cs

```diff
@@ -19,16 +19,35 @@ public static void Run(params string[] outputDirectories)
             var assemblies = new List<Assembly>();
             assemblies.Add(typeof(Instrumentation).Assembly);
 
-            // find all methods in Datadog.Trace.ClrProfiler.Managed.dll with [InstrumentMethod]
+            // Extract all InstrumentMethodAttribute at assembly scope level
+            var assemblyInstrumentMethodAttributes = from assembly in assemblies
+                                                     let attributes = assembly.GetCustomAttributes(inherit: false)
+                                                            .Select(a => a as InstrumentMethodAttribute)
+                                                            .Where(a => a != null).ToList()
+                                                     from attribute in attributes
+                                                     let callTargetClassCheck = attribute.CallTargetType ?? throw new NullReferenceException("The usage of InstrumentMethodAttribute in assembly scope must define the CallTargetClass property.")
```

Comment:
Can you include the type name of the attribute and maybe the target type / target method name strings in the exception message to make it easier to identify the attribute that was not constructed correctly?

Reply:
Or maybe we can just define a `ToString()` implementation and use that?

Reply:
Done in 7189594

Reply:
Thanks!